connect-inject/handler.go

```diff
@@ -371,7 +371,7 @@ func (h *Handler) Mutate(req *v1beta1.AdmissionRequest) *v1beta1.AdmissionRespon
 	// all patches are created to guarantee no errors were encountered in
 	// that process before modifying the Consul cluster.
 	if h.EnableNamespaces {
-		if err := namespaces.EnsureExists(h.ConsulClient, h.consulNamespace(req.Namespace), h.CrossNamespaceACLPolicy); err != nil {
+		if _, err := namespaces.EnsureExists(h.ConsulClient, h.consulNamespace(req.Namespace), h.CrossNamespaceACLPolicy); err != nil {
```

Comment:
`EnsureExists` now returns `(bool, err)` so we can log when a namespace gets created (vs already existed)